cpp/ql/test/library-tests/c++_exceptions/unreachable.expected

```diff
@@ -1,2 +1,2 @@
-| exceptions.cpp:25:13:30:9 | ExprStmt |
+| exceptions.cpp:25:13:25:19 | ExprStmt |
```

Comment:
The location of this `ExprStmt` appears to have mysteriously changed (could it be a synthesized location???).  The new one looks better than the old one.